docs/advanced-usage/using-proxies.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Proxies
+description: Learn how to proxy the Clerk Frontend API through your domain.
+---
+
+# Proxying the Clerk Frontend API
+
+Clerk supports two methods for configuring the Clerk Frontend API: CNAME and Proxy. 
```

Comment:
```suggestion
Clerk supports two configuration methods for connecting to the Clerk Frontend API: CNAME and Proxy. 
```
Please feel free to use any copy you see fit, I just thought that we're not actually configuring FAPI, we're just configuring how we'll connect to it. :/

Reply:
Maybe add a link to the DNS set up guide? (if there is one...) :laughing: